js/app.js

```diff
@@ -10,7 +10,7 @@
 // `Vue.extend()`, or just a component options object.
 // We'll talk about nested routes later.
 const routes = [
-    { path: "/course", component: Courses },
+    { path: "/", component: Courses },
```

Comment:
What happens if someone goes to `/course`? What route does it default to? There should be a way to set a default route, perhaps.

Reply:
I can create a 404 error component